libcore/gof-callwhenready.vala

```diff
@@ -28,11 +28,11 @@ public class GOF.CallWhenReady : Object {
 
 
     public CallWhenReady (GLib.List<GOF.File> _files, call_when_ready_func? _f = null) {
-        files = _files.copy ();
+        files = _files.copy_deep ((GLib.CopyFunc) GOF.File.@ref);
```

Comment:
This does not compile with the new gof-file.vapi.  Can we use Object.ref?   Or add back ref() and unref() to .vapi?

Reply:
I'm going to use Object.ref here as GOF.File.ref is just a wrapper (that will disappear when the Vala port will be completed)